pkgs/top-level/python-packages.nix

```diff
@@ -29288,4 +29288,47 @@ in modules // {
       sha256 = "14kb9gxw39zhvrijhp066b4bm6bgv35iw56c394y4dyczpha0dij";
     };
   };
+
+  sqlalchemy_bb8 = buildPythonPackage rec {
```

Comment:
as we're actually trying to get rid off these older versions, I suggest you put these inside the expression of buildbot